traversability_estimation/include/traversability_estimation/TraversabilityEstimation.hpp

```diff
@@ -155,6 +161,20 @@ class TraversabilityEstimation
    */
   bool requestElevationMap(grid_map_msgs::GridMap& map);
 
+  /*!
+   * Initializes a new traversability map based on the given grid map message. Previous traversability map is overwritten.
+   * @param message grid map message to be used to compute new traversability map.
+   * @return true on success.
+   */
+  bool initializeTraversabilityMapFromGridMapMessage(const grid_map_msgs::GridMap& message);
+
+  /*!
+   * Initializes a new traversability map based on the given grid map. Previous traversability map is overwritten.
+   * @param gridMap grid map object to be used to compute new traversability map.
+   * @return true on success.
+   */
+  bool initializeTraversabilityMapFromGridMapMessage(const grid_map::GridMap& gridMap);
```

Comment:
In general I prefer to be a bit more verbose and make sure that it's 100% clear what the method does, based on its name. I would not name it `initializeTraversabilityMap` because it's not clear how you init the traversability map (yes, since you pass the param `gridMap` then you could say that it's clear, but I'm not 100% satisfied). If you don't like this verbose name I can find something else ...

Reply:
Ok, I agree. But I would still name it `initializeTraversabilityMapFromGridMap`, since you are not initializing from a message.

Reply:
ah, yes, I forgot to change the name after I changed the type of param! :+1: